README.md

```diff
@@ -8,22 +8,25 @@ Testing after that is welcome, but will not be included in the official statisti
 Everyone, all contributions are welcome!
 
 ## How to contribute?
-The [Issues](https://github.com/osrf/ros2_test_cases/issues) page has several tickets each containing specific instructions to test a particular functionality of ROS 2 Jazzy Jalisco.
+The [Issues](https://github.com/osrf/ros2_test_cases/issues) page has several tickets each containing specific instructions to test a particular functionality of ROS 2 Kilted Kaiju.
 Each ticket will have the following sections:
 
 1. **Setup**: Details on the desired hardware and software setup for this test. The following combinations are possible.
-   1. DDS vendor: `FASTDDS`, `CYCLONEDDS` or `CONNEXTDDS`
+   1. RMW vendor: `fastdds`, `cyclonedds`, `connextdds` or `zenoh`
    2. BuildType:
-      * `binary`: pre-built packaged ROS 2 workspace from [the releases page](https://github.com/ros2/ros2/releases/tag/release-jazzy-beta-20240430)
+      * `binary`: pre-built packaged ROS 2 workspace from [the releases page](https://github.com/ros2/ros2/releases/tag/release-kilted-beta-20240430)
```

Comment:
@mjcarroll @cottsay we need to update this link once the pre-release tarballs are uploaded.

Reply:
Beta release is published.

```suggestion
      * `binary`: pre-built packaged ROS 2 workspace from [the releases page](https://github.com/ros2/ros2/releases/tag/release-kilted-beta-20250430)
```